azure-mgmt-containerservice/pom.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-parent</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>azure-mgmt-containerservice</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Microsoft Azure SDK for Container Service Management</name>
```

Comment:
should we use azure kubernetes service now?

Reply:
The Container Service uses the name of the provider. And it include not only kubernetes, but also container service. I think if we want to name it to kubernetes, we need to separate these two service.